classes/console/console.gd

```diff
@@ -236,12 +236,16 @@ func enable(enabled):
 	if visible:
 		input_line.grab_focus()
 		input_line.text = ""
+		_on_Input_text_changed("")
```

Comment:
Calling callbacks outside of signals seems like bad practice - is there no other way this could be dealt with?

Reply:
I mean I could move the code inside of the callback into a seperate function and then invoke that from the callback and here.

But that just seems more effort than it seems worth, and the callback is relating on input of the text field changing -which is exactly what is happening here. That's why I invoked it, as there's some logic which needs to run when the text changes, so it all just feels appropriate.

Reply:
Hm. Maybe do it by emitting the text_changed signal instead, so it's working as expected for the node and we don't get any unexpected behavior?

Reply:
Didn't know that was a thing! If you could make that change, much appreciated (not on pc rn)